### PR TITLE
feat: cli to pause all connectors of WS

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

We have some churned workspaces that need to have their connectors paused.

### CODE REVIEW

**Review by spolu — APPROVED** [review]

File: types/src/front/lib/connectors_api.ts [hunk 1]
```diff
@@ -531,6 +531,9 @@ export class ConnectorsAPI {
   private async _resultFromResponse<T>(
     response: Response
   ): Promise<ConnectorsAPIResponse<T>> {
+    if (response.status === 204) {
```
Comment (spolu) [inline types/src/front/lib/connectors_api.ts:534]: Mind adding a small comment here?

Comment (fontanierh) [reply types/src/front/lib/connectors_api.ts:534]: def

File: front/admin/cli.ts [hunk 2]
```diff
@@ -78,6 +80,36 @@ const workspace = async (command: string, args: parseArgs.ParsedArgs) => {
...
+      const connectorsApi = new ConnectorsAPI(logger);
+      for (const connectorId of connectorIds) {
+        console.log(`Pausing connectorId=${connectorId}`);
+        const res = await connectorsApi.pauseConnector(connectorId);
+        if (res.isErr()) {
+          throw new Error(res.error.message);
+        }
+      }
+
+      console.log("Connectors paused");
```
Comment (spolu) [inline front/admin/cli.ts:109]: nit

Comment (fontanierh) [reply front/admin/cli.ts:109]: it's actually on purpose we're in cli

**Pull request merged**